requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pip==23.0
+pip~=22.0
```

Comment:
Want to know the reason why we use ~=22.0 here instead of 23.0

Reply:
The latest version of the 23.x branch causes issues with pip-installed binaries being available and usable on path. I downgraded the previous major release for now, until we can identify the best path around this.